ui/index.html

```diff
@@ -1,18 +1,19 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" name="viewport" content="width=device-width">
+    <meta charset="utf-8" name="viewport" content="width=device-width" />
     <title>Tetra</title>
```

Comment:
Maybe we should even update the title to what it really is now :smi

Reply:
Sure! I wanted to keep the UI related changes to a minimum (just in case someone else wants to spend some time doing ui work 😉), but that is well within the limits of a drive by 😄